src/tests/test_import.py

```diff
@@ -3,11 +3,17 @@
 """Test the import statement."""
 
 import pytest
-
+import sys
 
 def test_relative_missing_import():
     """Test that a relative missing import doesn't crash.
     Some modules use this to check if a package is installed.
     Relative import in the site-packages folder"""
     with pytest.raises(ImportError):
         from . import _missing_import
+
+
+def test_import_all_on_second_time():
```

Comment:
I don't see any test here @amos402 ?

Reply:
oh, I see the actual test is in import, might be worth adding a comment @amos402 !

Reply:
anyway please also add a docstring with explanation of this unit test, like you described in the PR.

Reply:
Got it.